armory/scenarios/poisoning_witches_brew.py

```diff
@@ -340,12 +342,49 @@ def load_dataset(self, eval_split_default="test"):
         self.i = -1
 
     def load_metrics(self):
-        self.non_trigger_accuracy_metric = metrics.MetricList("categorical_accuracy")
-        self.trigger_accuracy_metric = metrics.MetricList("categorical_accuracy")
+        self.hub.connect_meter(
+            Meter(
+                "mean_accuracy_non_trigger_images",
+                metrics.get_supported_metric("categorical_accuracy"),
+                "scenario.y[benign]",
+                "scenario.y_pred[benign]",
+                final=np.mean,
+                final_name="mean_accuracy_non_trigger_images",
+                record_final_only=False,
+            )
+        )
+        self.hub.connect_meter(
+            Meter(
+                "accuracy_trigger_images",
+                metrics.get_supported_metric("categorical_accuracy"),
+                "scenario.y[adversarial]",
```

Comment:
Can we pick any name for the context/stage?  I think trigger/non-trigger is better than adversarial/benign, because it's ambiguous whether adversarial refers to the poisoned training images or the unpoisoned trigger images

Reply:
Yes, we can.

Reply:
I updated those stages in the latest push.

Reply:
They are now "trigger"/"non-trigger"